src/main/java/uk/gov/hmcts/ccd/domain/model/definition/RoleAssignment.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.ccd.domain.model.definition;
```

Comment:
please remove `definition`. In data store `definition` is anything related to our Definition Store which returns CCD definition information. This would make it very confusing